views/forms/ChartForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import Button from '@material-ui/core/Button'
+
+import Form from './Form'
+import BarChartFields from './BarChartFields'
+
+const ChartForm = ({ classes, handleSubmit }) => {
+  const [formValues, setFormValues] = useState({
+    title: '',
+    assessment: '',
+    variable: '',
+    fieldLabelValueMap: [
+      {
+        value: '',
+        label: ''
+      }
+    ]
+  })
+
+  const updateFormValues = (e,) => setFormValues({ ...formValues, [e.target.name]: e.target.value })
+
+  return (
+    <Form handleSubmit={(e) => handleSubmit(e, formValues)}>
+      <BarChartFields 
+        classes={classes} 
+        updateFormValues={updateFormValues} 
```

Comment:
instead of passing `updateFormValues` and `setFormValues` what do you think about only passing `setFormValues`?

Reply:
[ab607bc](https://github.com/AMP-SCZ/dpdash/pull/84/commits/ab607bcdc9895d5c9da30b9ab50689aab6b3b00e)